### PR TITLE
[www] Update copyright years in footer 

### PR DESCRIPTION
**Pull request by imphil**
It's not 2019 any more (I wouldn't mind it, though).

Dynamically update the copyright year to include the current year
("2019" becomes "2019 - 2020").

### CODE REVIEW

**Comment by cdgori** [comment]
You might want a lawyer to double-check that, in case it's important.  In some circumstances, I was told to change that to "2019- $currentyear" for reasons (that I don't recall any longer).

**Comment by sjgitty** [comment]
I believe we've gotten legal OK to remove all years in past scrubs. But I'll run this by our friends.

File: site/docs/layouts/partials/footer.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
     </ul>
   </nav>
 
-  <p>© 2019 <a href="https://www.lowrisc.org">lowRISC</a> contributors, <a
+  <p>© 2019 &ndash; {{ now.Format "2006" }} <a href="https://www.lowrisc.org">lowRISC</a> contributors, <a
```
Comment (imphil) [inline site/docs/layouts/partials/footer.html:39]: Looks odd, but it's go.

**Comment by imphil** [comment]
Updated to use the current year as "end year" instead, as per legal advice.

**Pull request merged**